### ISSUE TITLE
issue

### ISSUE THREAD

**Issue by skidifyings** [cliphd/TTNC#6]
i'm legit checking 8ls and they r unavailable when they aren't 💀

**Comment by cliphd** [comment]
I've heard that it's been patched. I tried it myself and the same thing happened. I'll update the checker when I find a fix.

**Comment by Al3xlol** [comment]
Yes 90% of checkers don’t work great right now due to update in end points tiktok changed them rip it’s really hard fix
